lightkurve/data/short_cadence_month_lookup.csv

```diff
@@ -0,0 +1,50 @@
+Quarter,Month,Date
```

Comment:
What is the exact meaning of `Date`?  I suggest calling it e.g. `StartTime` if that is what it is?

Reply:
I assume that date string is a starttime, we can change it to that.